addons/l10n_in_edi/tests/test_edi_json.py

```diff
@@ -31,6 +32,28 @@ def setUpClass(cls, chart_template_ref="l10n_in.indian_chart_template_standard")
             "l10n_in_gst_treatment": "regular",
         })
         cls.product_a.write({"l10n_in_hsn_code": "01111"})
+        cls.product_a2 = cls.env['product.product'].create({
+            'name': 'product_a2',
+            'uom_id': cls.env.ref('uom.product_uom_unit').id,
+            'lst_price': 1000.0,
+            'standard_price': 1000.0,
+            'property_account_income_id': cls.copy_account(cls.company_data['default_account_revenue']).id,
+            'property_account_expense_id': cls.copy_account(cls.company_data['default_account_expense']).id,
+            'taxes_id': [(6, 0, cls.tax_sale_a.ids)],
+            'supplier_taxes_id': [(6, 0, cls.tax_purchase_a.ids)],
+            "l10n_in_hsn_code": "01111",
```

Comment:
It would be good to have lines with different hsn codes as well?  (is there a case where you do not have an hsn code?)

Reply:
if there is no HSN code then it's a configuration mistake and raises an error in _check_move_configuration so it's always there, okay I add it.

Reply:
I add a different HSN code product